renderdoc/replay/replay_controller.cpp

```diff
@@ -1518,7 +1516,19 @@ bool ReplayController::SaveTexture(const TextureSave &saveData, const char *path
         exrImage.width = td.width;
         exrImage.height = td.height;
         exrHeader.pixel_types = pixTypes;
-        exrHeader.requested_pixel_types = reqTypes;
+
+        if(saveFmt.compByteWidth == 4)
+        {
+          int reqTypes[4] = {TINYEXR_PIXELTYPE_FLOAT, TINYEXR_PIXELTYPE_FLOAT,
```

Comment:
Once this array goes out of scope at the end of the if, `exrHeader.requested_pixel_types` is pointing to dangling memory, which may or may not still have valid data.

I think instead you should leave the `reqTypes` array declaration where it is and instead do a loop over all indices setting the enum to `TINYEXR_PIXELTYPE_FLOAT`.